test/Information_Gain_test.jl

```diff
@@ -0,0 +1,53 @@
+using Random
+using Statistics
+using StatsBase  # Importing StatsBase for the countmap function..
+using Test
+
+# Import the functions from your Information Gain implementation..
+include("..\\Information_Gain_Arka.jl")
```

Comment:
you can add this to run_tests.jl: 
@testset "DecisionTreeAndRandomForest.jl" begin
    include("Information_Gain_test.jl")
end